srcpkgs/opencv/template

```diff
@@ -1,10 +1,12 @@
 # Template file for 'opencv'
 pkgname=opencv
-version=3.4.3
-revision=2
+version=3.4.4
+revision=1
 build_style=cmake
 configure_args="-DENABLE_PRECOMPILED_HEADERS=OFF -DWITH_OPENMP=ON
- -DWITH_OPENCL=ON -DENABLE_CXX11=ON"
+ -DWITH_OPENCL=ON -DENABLE_CXX11=ON -DOPENCV_SKIP_PYTHON_LOADER=ON
+ -DOPENCV_PYTHON3_INSTALL_PATH=/usr/lib/python${py3_ver}/site-packages
```

Comment:
can't this be replaced by py3_sitelib and py2_sitelib ?

Reply:
thanks for the hint, wasn't aware this exists